static/src/stylesheets/module/identity/_manage-account-tab.scss

```diff
@@ -473,3 +473,46 @@
         }
     }
 }
+
+.manage-account-flexrow {
+    @include mq(tablet) {
+        display: flex;
+    }
+}
+.manage-account-flexrow_item {
+    flex: 1 1 auto;
+    &:not(:last-child) {
+        margin-right: $gs-gutter;
+    }
+}
+.manage-account-flexrow_item--static {
+    flex: 0 0 auto;
+}
+
+.manage-account-ty {
+    display:grid;
+    grid-column-gap: $gs-gutter * 4;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    margin: 0 $gs-gutter * 2;
+    align-items: center;
+}
+
+.manage-account-ty__title {
+    @include fs-headline(8);
+    text-align: right;
+    font-size: 10em;
+    line-height: .9em;
+    grid-column: 1 / 3;
+    grid-row: 1 / 3;
+    color: $maps-main-1;
+}
+
+.manage-account-ty__content {
+    grid-column: 3;
```

Comment:
Do you have a proper browser fallback in place?

Reply:
yes, it just stacks like normal text BUT i need to also make it work like that on mobile